### ISSUE TITLE
Downloading a custom package contains not-minified css-files without root-container-rule

### ISSUE THREAD

**Issue by 0xnoob** [highlightjs/highlight.js#3231]
**Describe the issue/behavior that seems buggy**
Downloading a custom package contains the non-minified CSS-files and the The `base16` folder is empty.  
They are also missing this common rule:
```css
.hljs {
  display: block;
  overflow-x: auto;
  padding: 0.5em;
}
```
[The docs says:](https://highlightjs.readthedocs.io/en/latest/style-guide.html#typography-and-layout-dos-and-don-ts)
>If your theme exists within the core project you do not need to add these rules by hand as they are added to all themes by default during the build process.

Maybe something went wrong in the build process?

**Sample Code or Instructions to Reproduce**
 1. goto https://highlightjs.org/download/
 2. click [download]
 3. check the zip file for the above mentioned issues.

**Expected behavior**
The CSS-files should be minified, should include the common rule and the `base16` folder shouldnt' be empty.

**Comment by joshgoebel** [comment]
Yes, looks like this broke with the v11 release.  For now I'd suggest getting the rest of the files you need from our CDN release: https://github.com/highlightjs/cdn-release/tree/11-stable
